### DISCUSSION TITLE
Looking to contribute: Add ellipsis for the label fields

### DISCUSSION THREAD

**Discussion by navaneeth-dev** [pocketbase/pocketbase#3779] [Q&A] [answered]
I might be able to contribute to this. Can you elaborate what you need?

So in label text when it overflows you need ellipsis right?

https://github.com/orgs/pocketbase/projects/2/views/1?pane=issue&itemId=44879789

Thanks in advance

**Comment by ganigeorgiev** [comment, marked as answer]
I've actually already implemented it locally as part of the logs UI refactoring.
